assets/app/view/game/buy_trains.rb

```diff
@@ -86,10 +86,12 @@ def render
 
         if (round.can_buy_train? && round.corp_has_room?) || round.must_buy_train?
           children << h(:h3, h3_props, 'Available Trains')
-          children << h(:div, div_props, [
-            *from_depot(depot_trains),
-            *other_corp_trains.any? ? other_trains(other_corp_trains) : '',
-          ])
+          lines = [*from_depot(depot_trains)]
```

Comment:
this is redundant, just do lines = from_depot

Reply:
No longer relevant as code changed.